setup.py

```diff
@@ -26,7 +21,13 @@ def get_requirements():
             "Programming Language :: Python :: 2.7",
             "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
       ],
-      install_requires=get_requirements(),
+      install_requires=[
```

Comment:
is this change really necessary? I kinda like requirements.txt files.

Reply:
Well, Travis can't seem to find the file on Python 2.6 or 3.6 for some reason so seems necessary for now, at least.